.circleci/config.yml

```diff
@@ -688,59 +767,64 @@ workflows:
   version: 2
   go-tests:
     jobs:
-      - check-vendor:
+      - check-vendor: &filter-ignore-non-go-branches
           filters:
             branches:
               ignore:
                 - stable-website
                 - /^docs\/.*/
                 - /^ui\/.*/
-      - lint-consul-retry:
+      - lint-consul-retry: *filter-ignore-non-go-branches
+      - lint: *filter-ignore-non-go-branches
+      - test-connect-ca-providers: *filter-ignore-non-go-branches
+      - dev-build: *filter-ignore-non-go-branches
+      - go-test:
+          requires: [dev-build]
+      - go-test-api:
+          requires: [dev-build]
+      - go-test-sdk: *filter-ignore-non-go-branches
+  build-distros:
+    jobs:
+      - check-vendor: *filter-ignore-non-go-branches
+      - build-386: &require-check-vendor
+          requires:
+            - check-vendor
+      - build-amd64: *require-check-vendor
+      - build-arm: *require-check-vendor
+      # every commit on ui-staging and master will have a rebuilt UI
+      - frontend-cache:
           filters:
             branches:
-              ignore:
-                - stable-website
-                - /^docs\/.*/
-                - /^ui\/.*/
-      - go-fmt-and-vet:
+              only:
+                - master
+                - ui-staging
```

Comment:
This branch no longer exists, but probably doesn't matter because `release/1.7.x` doesn't control the flow from `master` anyway, and we just care about it matching `/release\/\d+\.\d+\.x$/` to trigger `publish-static-assets`?